Gallery/Contours/NCL_coneff_11.py

```diff
@@ -0,0 +1,127 @@
+"""
+NCL_coneff_11.py
+================
+
+This script illustrates the following concepts:
+   - Filling contours with multiple shaded patterns
+   - Filling contours with stippling (solid dots)
+   - Changing the size of the dot fill pattern in a contour plot
+   - Overlaying a stipple pattern to show area of interest
+   - Changing the density of contour shaded patterns
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/coneff_11.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/coneff_11_1_lg.png
+                          https://www.ncl.ucar.edu/Applications/Images/coneff_11_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+import geocat.datafiles as gdf
+import geocat.viz as gv
+
+###############################################################################
+# Read in data
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get('netcdf_files/atmos.nc'), decode_times=False)
+
+# Select meridional wind at lowest pressure level
+v = ds.V.isel(time=0, lev=0)
+
+###############################################################################
+# Plot
+
+# Generate figure (set its size (width, height) in inches)
+plt.figure(figsize=(8, 10))
+ax = plt.axes()
+
+# Choose hatches to fill the contours
+hatches = ['/////', '/////', '/////', '/////', None, '..', '.', '.', '.']
```

Comment:
The original plot has different densities of the dots for the different regions. Can you see if there is a way to do that? It's okay if the densities in matplotlib are different from NCL. As long as we're showing the different options, I think it'll be good.

Reply:
The densities are different in Matplotlib, and unfortunately have a large increase/decrease in density with each change in level. The '...' level almost looks like a solid color, so I had to stick to just the '..' and '.' options.

Reply:
Ah that's understandable. I'll go ahead and approve this then